content/enterprise/install/interactive/installer.mdx

```diff
@@ -153,7 +153,7 @@ As of version 201902-01, TLS versions 1.0 and 1.1 are no longer supported in Ter
 
 #### TLS Ciphersuites
 
-Terraform Enterprise uses a standard set of ciphersuites by default. If necessary, a custom ciphersuite can be provided:
+Terraform Enterprise uses the `HIGH:!aNULL:!MD5` ciphersuites by default. If necessary, a custom ciphersuite can be provided:
```

Comment:
Wanted to make this more active voice. Let me know if they aren't the ones providing this!

```suggestion
Terraform Enterprise uses the `HIGH:!aNULL:!MD5` ciphersuites by default. You can also provide a custom ciphersuite if necessary. 
```